rust/Earthfile

```diff
@@ -60,8 +81,8 @@ get-tomljson:
     FROM alpine:3.18.3
     ARG USERARCH
     ARG version=2.1.0
-    RUN wget -O tomljson.tar.xz https://github.com/pelletier/go-toml/releases/download/v${version}/tomljson_${version}_linux_${USERARCH}.tar.xz && \
-        tar -xf tomljson.tar.xz && \
+    RUN wget -O tomljson.tar.xz https://github.com/pelletier/go-toml/releases/download/v${version}/tomljson_${version}_linux_${USERARCH}.tar.xz; \
+        tar -xf tomljson.tar.xz; \
```

Comment:
is this change intentional or possibly this PR didn't included latest main (with Alex's changes)?

Reply:
it wasn't, thanks!

Reply:
Is this about `&&` vs `;` or something else? 

In the absence of `set -e` I think `&&` ought to be preferred.